masonry_winit/src/event_loop_runner.rs

```diff
@@ -45,46 +45,34 @@ impl From<accesskit_winit::Event> for MasonryUserEvent {
     }
 }
 
+/// The state of the current app.
 #[expect(unnameable_types, reason = "TODO")]
-#[allow(
-    clippy::large_enum_variant,
-    reason = "we don't have that many instances of it"
-)]
-pub enum WindowState {
-    Uninitialized(WindowAttributes),
-    Rendering {
-        handle: Arc<WindowHandle>,
-        accesskit_adapter: Adapter,
-    },
-    Suspended {
-        handle: Arc<WindowHandle>,
-        accesskit_adapter: Adapter,
-    },
-}
-
-impl Debug for WindowState {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        match self {
-            Self::Uninitialized(attrs) => f.debug_tuple("Uninitialized").field(attrs).finish(),
-            Self::Rendering { .. } => f.debug_struct("Rendering").finish_non_exhaustive(),
-            Self::Suspended { .. } => f.debug_struct("Suspended").finish_non_exhaustive(),
-        }
-    }
+#[derive(Default, Debug)]
+pub enum AppState {
+    /// The app is currently in suspended state,
+    /// usually set during [`ApplicationHandler::suspended()`].
+    #[default]
```

Comment:
It's not clear why this Default implementation exists. Should we ever expect users to create an instance of this type?

Reply:
Not really actually, let me remove that (I was thinking it'll be more like a guide for users initially.)